src/client.rs

```diff
@@ -888,42 +920,31 @@ mod tests {
     #[meilisearch_test]
     async fn test_create_key(client: Client, description: String) {
         let expires_at = OffsetDateTime::now_utc() + time::Duration::HOUR;
-        let mut key = KeyBuilder::new(description.clone());
+        let mut key = KeyBuilder::new();
         key.with_action(Action::DocumentsAdd)
+            .with_name("test_create_key")
             .with_expires_at(expires_at.clone())
+            .with_description(&description)
             .with_index("*");
         let key = client.create_key(key).await.unwrap();
 
         assert_eq!(key.actions, vec![Action::DocumentsAdd]);
-        assert_eq!(key.description, description);
+        assert_eq!(&key.description, &Some(description));
         // We can't compare the two timestamp directly because of some nanoseconds imprecision with the floats
```

Comment:
Here I *think* we should use the unique string generation for the name of the key and hardcode the description to a random value